source/change-log/index.html.md.erb

```diff
@@ -13,22 +13,22 @@ description: Find out what has changed in the specification and make sure your a
 
  * Clarification on format for Gov-Client-Timezone
 
-This will be updated in the <a href="https://www.gov.uk/government/publications/direction-under-regulation-22-of-the-delivery-of-tax-information-through-software-ancillary-metadata-regulations-2019-si-2019360">Directions</a>.
+This will be updated in the Directions. To check what you need to send, <a href="../connection-method/" onclick="ga('send', 'event', 'On Page Navigation', 'Change Log', 'Select your connection method');">select your connection method</a>.
```

Comment:
I could be wrong but this ga function is deprecated and no longer supported. It went away when tracking consent was rolled out.

Reply:
We use it 26 times in the repo so should discuss separately. For now I put the same link to `select connection method` as in other places.